spec/support/expect_url_with_query.rb

```diff
@@ -0,0 +1,6 @@
+def expect_url_with_query(expected_path:, expected_query_params:)
```

Comment:
There is already a helper that does this `spec/support/helpers.rb#163` `expect_page_to_be_displayed_with_query`